staging/knative/charts/eventing/values.yaml

```diff
@@ -0,0 +1,2 @@
+global:
+  serviceLabels: {}
```

Comment:
what about having more available variables such as like this helm chart did at that time https://github.com/simplytunde/charts/blob/master/incubator/knative/values.yaml ?

Reply:
The whole `build` section is not needed, since we're not bundling `knative-build` in favor of the tekton pipeline. I don't see why the `eventing` section is necessary unless we have very specific needs to change the official images for the controller and webhook and the arguments.

Then, the remaining meaningful configuration options are:
1. Controlling if RBAC should be installed, which I think it should always be.
2. Setting up the nemaspace for just eventing. This might be handy if we want to configure it otherwise in konvoy. But for now I prefer keeping the default, since there are 3 namespaces involved, not just eventing, in the package.

Also, it seems to me that it would be more meaningful to add values for public configuration options in `ConfigMap`s, e.g.: https://github.com/knative/serving/blob/master/install/CONFIG.md

WDYT?

Reply:
FIY this is the `value.yaml` from the most recent helm chart PR for knative:
https://github.com/tomleah/helm-charts/blob/2afbc130a23ab9aae052967bb1bc20eb739a783a/incubator/knative/values.yaml
Unfortunately the PR became inactive so I created these charts here.